tests/api2/test_425_smb_protocol.py

```diff
@@ -605,6 +624,14 @@ def test_152_check_xattr_via_smb(request, xat):
     # Python base64 library appends a `\t` to end of byte string
     assert xat_bytes == bytes, str(err)
 
+    # Check via kernel client.
+    kcontent = get_stream('afp_xattr_testfile', AFPXattr[xat]['smbname'])
+    err = {
+        "name": xat,
+        "b64data": b64encode(kcontent[:-1])
+    }
+    assert xat_bytes == bytes, str(err)
```

Comment:
Shouldn't this check use `kcontent` rather than `xat_bytes`?

Reply:
Yes! (doh)